src/link/Elf.zig

```diff
@@ -1521,6 +1521,8 @@ fn linkWithLLD(self: *Elf, comp: *Compilation) !void {
                     } else {
                         break :o "crt1.o";
                     }
+                } else if (target.os.tag == .haiku) {
```

Comment:
Depending on which PR is merged first this conflicts with #8730 or the other way around.

Reply:
Thank you for the heads up on this. I will keep an eye out and handle updating the PR if needed.

Reply:
Thanks, both of you. #8730 is merged now. Once this is rebased I think it is ready to be merged. Nice progress on Haiku support!